dotcom-rendering/src/model/enhance-H2s.test.ts

```diff
@@ -1,5 +1,5 @@
 import { blockMetaData } from '../../fixtures/manual/block-meta-data';
-import { enhanceH2s } from './enhance-H2s';
```

Comment:
I think it would be easier to understand / change this code in the future if we leave the `getUnique` as an implementation detail inside the enhancer and then add a test here to verify what happens if I pass multiple h2s with the same title.

It's nicer if the tests execute at as higher level as possible, focussing on results and leaving it up to the things being tested to decide how they do things

Reply:
Absolutely, I'll refactor this.